corehq/apps/reports/views.py

```diff
@@ -521,9 +521,10 @@ def email_report(request, domain, report_slug, dispatcher_class=ProjectReportDis
         recipient_emails.add(request.couch_user.get_email())
 
     request_data = request_as_dict(request)
-
     report_type = dispatcher_class.prefix
-    send_email_report.delay(recipient_emails, domain, report_slug, report_type,
+    request_data['startdate'] = str(request_data['datespan'].startdate)
```

Comment:
Same here: `request_data['datespan'].startdate.isoformat()`. Personally would use `pop` instead of `del` but up to you